schemas/v1/payload.json

```diff
@@ -160,15 +160,6 @@
       "default": [],
       "description": "Example: `['/bin/bash', '-c', 'ls']`."
     },
-    "encryptedEnv": {
-      "title": "List of encrypted environment variable mappings.",
-      "description": "List of base64 encoded asymmetric encrypted environment variables. See /docs/reference/workers/docker-worker/environment#encrypted-environment-variables",
-      "type": "array",
-      "items": {
-        "title": "Base64 encoded encrypted environment variable object.",
-        "type": "string"
-      }
-    },
```

Comment:
I forgot we don't have `additionalProperties: false` here, so this actually *won't* generate an error if the parameter is still used.  Oh well :(

Reply:
That comes from our old try on adding this restriction and breaking a bunch of ill-formed task payloads.

Reply:
Yup, I understand :)

At least with this change the variable won't be in the docs anymore.